src/NServiceBus.AmazonSQS/Configure/EventToTopicsMappings.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    class EventToTopicsMappings
+    {
+        public void Add(Type eventType, string[] topicsNames)
+        {
+            if (!eventsToTopicsMappings.TryGetValue(eventType, out var mapping))
+            {
+                mapping = new HashSet<string>();
```

Comment:
If you specify the comparer in the ctor the below contains code is no longer necessary I think

Reply:
Not sure I understand @danielmarbach